hexrd/config/material.py

```diff
@@ -35,11 +42,16 @@ def active(self):
     @property
     def materials(self):
         """Return a dict of materials."""
+        #
+        # If reset_exclusions is False, we use the material as read from
+        # the file. This includes not using the dmin value, which may alter
+        # the HKLs available.
+        #
+        kwa = {"f": self.definitions}
+        if self.reset_exclusions:
+            kwa["dmin"] = valWUnit("dmin", "length", self.dmin, "angstrom")
```

Comment:
These three lines could be inside the `if not hasattr(self, '_materials')`, right?

Reply:
Did this.